core/src/main/java/com/twosigma/beaker/core/Main.java

```diff
@@ -96,6 +96,7 @@ private static CommandLine parseCommandLine(String[] args)
     opts.addOption(null, "require-password", false, "Ask for password when connecting");
     opts.addOption(null, "listen-interface", true, "Interface to listen on - requires ip address or '*'");
     opts.addOption(null, "portable", false, "Configuration and runtime files located in application instead of user home directory.");
+    opts.addOption(null, "nginx-bogus-logging", false, "Switchs off logging unsuccessfull requests from outdated beaker pages.");
```

Comment:
nginx-bogus-logging -> hide-zombie-logging

default should be true.

Switchs off logging unsuccessfull requests from outdated beaker pages.
->
Prevent distracting logging by Beaker clients of previous server instances.